tests/unit/flipswitch/flipswitch_core.js

```diff
@@ -3,60 +3,54 @@
+test( "select based flipswitch should have theme inherit", function( assert ) {
+	assert.hasClasses( $( "#flip-select-second-option" ).parent(), "ui-bar-inherit" );
 } );
-test( "checkbox based flipswitch should have theme inherit", function() {
-	ok( $( "#flip-checkbox-active" ).parent().hasClass( "ui-bar-inherit" ), "should be inherit theme" );
-} );
-test( "select based flipswitch should have theme inherit", function() {
-	ok( $( "#flip-select-second-option" ).parent().hasClass( "ui-bar-inherit" ), "should be inherit theme" );
-} );
-test( "checkbox based flipswitch is toggled on click", function() {
-	ok( !$( "#flip-checkbox" ).parent().hasClass( "ui-flipswitch-active" ), "should not be active" );
+test( "checkbox based flipswitch is toggled on click", function( assert ) {
+	assert.lacksClasses($( "#flip-checkbox" ).parent(), "ui-flipswitch-active" );
 	$( "#flip-checkbox" ).parent().click()
-	ok( $( "#flip-checkbox" ).parent().hasClass( "ui-flipswitch-active" ), "should be active" );
+	assert.hasClasses( $( "#flip-checkbox" ).parent(), "ui-flipswitch-active" );
 } );
-test( "select based flipswitch is toggled on click", function() {
+test( "select based flipswitch is toggled on click", function( assert ) {
 	$( "#flip-select" ).click();
-	ok( $( "#flip-select" ).parent().hasClass( "ui-flipswitch-active" ), "should be active" );
+	assert.hasClasses( $( "#flip-select" ).parent(), "ui-flipswitch-active" );
 } );
-test( "checkbox based flipswitch is not active after left swipe", function() {
+test( "checkbox based flipswitch is not active after left swipe", function( assert ) {
 	$( "#flip-checkbox" ).trigger( "swipeleft" );
-	ok( !$( "#flip-checkbox" ).parent().hasClass( "ui-flipswitch-active" ), "should be active" );
+	assert.lacksClasses( $( "#flip-checkbox" ).parent(), "ui-flipswitch-active" );
 } );
-test( "select based flipswitch is not active after left swipe", function() {
+test( "select based flipswitch is not active after left swipe", function( assert ) {
 	$( "#flip-select" ).trigger( "swipeleft" );
-	ok( !$( "#flip-select" ).parent().hasClass( "ui-flipswitch-active" ), "should be active" );
+	assert.lacksClasses( $( "#flip-select" ).parent(), "ui-flipswitch-active" );
 } );
-test( "checkbox based flipswitch is active after right swipe", function() {
+test( "checkbox based flipswitch is active after right swipe", function( assert ) {
 	$( "#flip-checkbox" ).trigger( "swiperight" );
-	ok( $( "#flip-checkbox" ).parent().hasClass( "ui-flipswitch-active" ), "should not be active" );
+	assert.hasClasses( $( "#flip-checkbox" ).parent(), "ui-flipswitch-active" );
 } );
-test( "select based flipswitch is active after right swipe", function() {
+test( "select based flipswitch is active after right swipe", function( assert ) {
 	$( "#flip-select" ).trigger( "swiperight" );
-	ok( $( "#flip-select" ).parent().hasClass( "ui-flipswitch-active" ), "should not be active" );
+	assert.hasClasses( $( "#flip-select" ).parent(), "ui-flipswitch-active" );
```

Comment:
What i mean is keep all the assertions just logically group them inside fewer `test()` calls.